classes/game.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './item'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(publish_date, multiplayer, last_played_at)
+    super(publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  def can_be_archived?
+    super && (Date.today.year - @last_played_at.to_i) > 2
+  end
```

Comment:
- [ ] As I mentioned in the previous comment and to correctly match the project requirements, kindly make sure you update the above method code in the way it also includes `months` and `days` in calculations. Forr example:
```ruby
  def can_be_archived?
      today = Date.today
      two_years_ago = Date.new(today.year - 2, today.month, today.day)

      super && @last_played > two_years_ago
    end
```

Reply:
resolved